neuron/src/app/Neuron/CLI/Types.hs

```diff
@@ -145,7 +145,7 @@ commandParser defaultNotesDir now = do
           <|> fmap
             (const . Some . IDSchemeCustom)
             (option str (long "id" <> help "Use a custom ID" <> metavar "IDNAME"))
-      pure $ New $ NewCommand title format now (idSchemeF day) edit
+      pure $ New $ NewCommand title format (LocalTime dayParam midday) (idSchemeF dayParam) edit
```

Comment:
We shouldn't be automatically filling in the time at all, if the user has not specified one. 

I think there shoud be a `neuron new --date=foo` (replacing `--day`) where "foo" can be either date or a datetime. You can reuse the `DateMayTime` type alias here, and use `Either` to capture the sum.

Reply:
doing this, options-lib displays this default though:
![option](https://user-images.githubusercontent.com/4367963/90441346-16025c80-e0d9-11ea-9da1-aa893f718c37.png)

Reply:
`      dateParam <-
        option dateReader $
          long "date"
            <> metavar "DATE"
            <> value (Left today)
            <> showDefault
            <> help "Zettel creation date/time"
`

Reply:
I guess I have to change "showDefault" ?

Reply:
Ok, have foung it: `showDefaultWith`